modules/nw-metallb-configure-return-traffic-proc.adoc

```diff
@@ -73,6 +73,9 @@ spec:
       - ip-to: 10.132.0.0/14
         priority: 998
         route-table: 254
+      - ip-to: 169.254.0.0/17
```

Comment:
(a reminder  from the bug), this needs to be `169.254.169.0/29` (so the cherry-pick can't be clean)